.cloudbuild/prombench.yaml

```diff
@@ -0,0 +1,335 @@
+# https://cloud.google.com/build/docs/build-config-file-schema#options
+options:
+  env:
+  - 'CANCEL_BENCHMARK=false'
+  - 'CLUSTER_NAME=prombench'
+  - 'CREATE_CLUSTER=true'
+  - 'DASHBOARD_ID=$_DASHBOARD_ID'
+  - 'DOMAIN_NAME='
+  - 'GITHUB_STATUS_TARGET_URL='
+  - 'GKE_PROJECT_ID=$_GKE_PROJECT_ID'
+  - 'GRAFANA_ADMIN_PASSWORD=admin'
+  - 'LAST_COMMIT_SHA='
+  - 'OAUTH_TOKEN='
+  - 'PROMBENCH_IMAGE=$_PROMBENCH_IMAGE'
+  - 'PROMETHEUS_IMAGE_REPOSITORY=gke.gcr.io/prometheus-engine/prometheus'
+  - 'PROVIDER=gke'
+  - 'SERVICEACCOUNT_CLIENT_EMAIL=$_SERVICEACCOUNT_CLIENT_EMAIL'
+  - 'WH_SECRET='
+  - 'ZONE=us-east4-a'
+  substitutionOption: ALLOW_LOOSE
+  dynamicSubstitutions: true
+
+# https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values
+substitutions:
+  _COMMENTER: $(body.comment.user.login)
+  _COMMENTER_ROLE: $(body.comment.author_association) # OWNER / MEMBER / etc. (https://docs.github.com/en/graphql/reference/enums#commentauthorassociation)
+  _GIT_REPO: $(body.repository.full_name) # GoogleCloudPlatform/prometheus
+  _PR_COMMENT_ACTION: $(body.action)
+  _PR_COMMENT_TEXT: $(body.comment.body)
+  _PR_NUMBER: $(body.issue.number)
```

Comment:
Since the current set-up is not based on the [GitHub repo](https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen), the variables get substituted at runtime. Once this PR gets merged, we can change these variables to use the default available substitutions (maybe as a next iteration).

Reply:
ignore _PR_NUMBER only available for PR triggers

Reply:
also ignor github repo. Only project_id can be subbed